app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import { find, isNil, uniqBy } from 'lodash'
+
+import FormGrid from '~/components/molecules/FormGrid'
+import FormGroupTextValue from '~/components/atoms/FormGroupTextValue'
+import { RadioGroup, Radio, Text } from 'react-form';
+import { getAMIAction } from '~/components/supplemental_application/actions'
+import { formatPercent } from '~/utils/utils'
+
+const YesNoRadioGroup = ({field}) => {
+  return (
+    <div className='radio-group-inline'>
+      <RadioGroup field={field}>
+        {(group) => (
+          <React.Fragment>
+            <p className="radio-inline">
+              <Radio group={group} value={`true`} id={`${field}-yes`}/>
+              <label className="radio-inline_label" htmlFor={`${field}-yes`}>Yes</label>
+            </p>
+            <p className="radio-inline">
+              <Radio group={group} value={`false`} id={`${field}-no`}/>
+              <label className="radio-inline_label" htmlFor={`${field}-no`}>No</label>
+            </p>
+          </React.Fragment>
+        )}
+      </RadioGroup>
+    </div>
+  )
+}
+
+const getAMI = ({numHousehold, chartName, chartYear, amis}) => {
+  let ami = find(amis, {'chartType': chartName, 'year': chartYear, 'householdSize': numHousehold})
+
+  if (!isNil(ami)) {
+    return ami.amount
+  }
+}
+
+const getAMIPercent = ({income, ami}) => {
+  if (!income) {
+    return "Enter HH Income"
+  }
+  if (isNil(ami)) {
+    return "Error: AMI unknown"
+  }
+  return formatPercent(income / Number(ami))
+}
+
+
+
+class ConfirmedHouseholdIncome extends React.Component {
+
+  state = {
+    amis: {},
+    amiCharts: []
+  }
+
+  componentDidMount() {
```

Comment:
We talked about this and you showed me some code, I think now I understand better.

calling `componentDidMount` does not look right. I feel it should be called in the top level component (the Page component) if you have time we can pair review this and you can walk me thru the review app.